### PR TITLE
MGMT-13420: Adding cleanup of next-step-runner before starting it.

### PR DESCRIPTION
**Pull request by tsorya**
[MGMT-13420](https://issues.redhat.com//browse/MGMT-13420): Adding cleanup of next-step-runner before starting it.
In 4.12 for unknown reason podman fails to cleanup next-step-runner on agent restart and --rm doesn't help. To workaround the problem we are adding cleanup before starting it.

### CODE REVIEW

File: src/commands/actions/next_step_runner.go [hunk 1]
```diff
@@ -12,6 +13,8 @@ import (
 	"github.com/openshift/assisted-service/models"
 )
 
+const containerName = "next-step-runner"
```
Comment (osherdp) [inline src/commands/actions/next_step_runner.go:16]: let's reuse this var on line 78, to make sure we don't change it only on one place (and forget the other place)

**Review by osherdp — CHANGES_REQUESTED** [review]

**Review by osherdp — APPROVED** [review]

**Comment by omertuc** [comment]
Please check if this also resolves https://issues.redhat.com/browse/MGMT-13183

**Comment by tsorya** [comment]
@omertuc seems like it should handle it

**Comment by tsorya** [comment]
/retitle MGMT-13427:Adding cleanup of next-step-runner before starting it

**Pull request merged**